transcom/mymove
# INT B-21029 rejected office users can resubmit

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21029)

## Summary

Requested office users were not able to re-request after being rejected. We did not implement any functionality to allow that, so HERE WE ARE SUCKAS.

This PR does the following:
- adds `OfficeUserStatus` enum and updated relevant code, not sure why I didn't add that in the first place
- updates the `office_user_creator` service object to first check if the office user exists and they have a previous status of `REJECTED` - if so, it fires off a function to update values and change the status to `REQUESTED`
- added and updated relevant tests
- updated requested office user form to have a set width to get rid of rendering issues (when it's real skinny & ugly) & fix error banner to span the width of the form

### How to test

1. Access office app
2. Request an account
3. Access admin app
4. Reject that user
6. Access office app
7. Request an account with same email but different data
8. Submit - should be successful
9. Access admin app
10. You should now see that requested user with updated data

## Screenshots
Video walkthrough of admin rejection -> office user resubmission
https://github.com/user-attachments/assets/79b96d68-0686-4a50-a5f8-88e2ca2895ec



Fixed error width
![Screenshot 2024-09-05 at 11 02 29 AM](https://github.com/user-attachments/assets/06c03e55-deeb-4b5f-9cb9-d26b98c0880c)
